File: week-1/fizz-buzz.py
 def fizzBuzz(self, n: int) -> List[str]:
        mylist=['Fizz','Buzz','FizzBuzz']
        outputlist=[]
        for x in range(1,n+1):
            
            if x%5==0 and i%3==0:
                outputlist.append(mylist[2]) 
            elif x%3==0:
                outputlist.append(mylist[0])
            elif x%5==0:
                outputlist.append(mylist[1])
        
            else:
                outputlist.append(str(x))
        return outputlist

File: Math, String/fizzbuzz.cpp
class Solution {
    public:
    vector<string> fizzBuzz(int n) {
                vector<string> list;
        for (int i = 1; i < n+1; i++) {
            if(i%15==0){
                list.push_back("FizzBuzz");
            }
            else if(i%3==0){
                list.push_back("Fizz");
            }
            else if(i%5==0){
                list.push_back("Buzz");
            }
            else{
                list.push_back(to_string(i));
            }
        }
        return list;
    }
};
